numpy/linalg/linalg.py

```diff
@@ -1336,46 +1345,75 @@ def svd(a, full_matrices=True, compute_uv=True):
 
     .. versionadded:: 1.8.0
 
-    Broadcasting rules apply, see the `numpy.linalg` documentation for
-    details.
+    The decomposition is performed using LAPACK routine ``_gesdd``.
+
+    SVD is usually described for the factorization of a 2D matrix :math:`A`.
+    The higher-dimensional case will be discussed below. In the 2D case, SVD is
+    written as :math:`A = U S V^H`, where :math:`A=` `a`, :math:`U=` `u`,
+    :math:`S=` `np.diag(s)` and :math:`V^H=` `vh`. The 1D array `s` contains
+    the singular values of `a` and `u` and `vh` are unitary. The rows of `vh`
+    are the eigenvectors of :math:`A^H A` and the columns of `u` are the
+    eigenvectors of :math:`A A^H`. For row `i` in `vh` and column `i` in `u`,
+    the corresponding eigenvalue is `s[i]**2`.
```

Comment:
Double backticks. The single backticks are for links or content following a directive.

Reply:
In other words, do not use single backticks for "i" either, as it is not something that can be linked to, but is a bit of code. Maybe say "i-th" column instead, which has the advantage of being english.

Reply:
I checked to make sure, but apart from directives, single backticks are used for generating links.

Reply:
Thank for clarifying this. It did not appear to me what to conventions were so far. Let me try to get this right.